guide/daily-spending-wallet/security.md

```diff
@@ -166,39 +162,21 @@ Bitcoin wallet users may feel hesitant about the privacy implications of your pr
 
 ### PIN protection
 
-A PIN is one of the most common and traditional methods for securing a mobile application. This setup requires that users create a 4-6 digit PIN that they must enter when opening their wallet application or viewing sensitive information, such as their recovery phrase.
-
-<div class="image-slide-gallery">
+A [PIN](https://en.wikipedia.org/wiki/Personal_identification_number) (Personal identification number) is one of the most common and traditional methods for securing a mobile application. This setup requires users to choose a 4-6 digit number. They must enter it when opening their wallet application, or viewing sensitive information, such as their recovery phrase.
```

Comment:
```suggestion
A [PIN](https://en.wikipedia.org/wiki/Personal_identification_number) (personal identification number) is one of the most common and traditional methods for securing a mobile application. This setup requires users to choose a 4-6 digit number. They must enter it when opening their wallet application, or viewing sensitive information, such as their recovery phrase.
```

Weird to capitalize just the word "personal". Need to pick a lane. Start each word with lowercase. Alternatively, title-case it to draw attention to the fact that the words form the acronym.

Reply:
Updated.